### ISSUE TITLE
Script not working on anonymous giveaways displayed separately

### ISSUE THREAD

**Issue by woctezuma** [petetnt/neogaf-monkeybot#10]
The script is not triggered by giveaways organized by an anonymous benefactor if displayed separately:
http://www.neogaf.com/forum/showpost.php?p=231237400&postcount=184

However, it works fine for the post viewed inside the thread:
http://www.neogaf.com/forum/showthread.php?p=231237400#post231237400

**Comment by woctezuma** [comment]
I found the culprit:

`            if (/showpost|showthread/.test(href) && modBotPosts.length) {`
should be
`            if (/showpost|showthread/.test(href) && allPosts.length) {`

**Comment by petetnt** [comment]
Nice catch, can you sent a PR?

**Comment by woctezuma** [comment]
I don't know how to update my fork to your latest master.

**Comment by petetnt** [comment]
Are you on `master` branch?

First add this repo as the upstream:

`git remote add upstream https://github.com/petetnt/neogaf-monkeybot`

Then fetch the changes:

```
git fetch origin upstream
```

Reset your `master` to match the upstream (note that you will loose any changes):

```
git reset --hard upstream/master
```

Alternatively you can rebase your changes on top of your master but you will keep your changes:

```
git rebase upstream/master
```

but that might cause some conflicts.

Good idea is to work on branches, for example taking the master and then doing `git checkout -b fix-anon-giveaways` and then work there. When you are ready, you can do `git push origin fix-anon-giveaways` and start the normal PR things. When the thing gets merged, you can just `git checkout master` and pull the changes in `git pull upstream master`

**Comment by woctezuma** [comment]
Thanks. The problem was I had committed some changes to `master` but they were mistakes, I thought I was on the branch when I was not. I am going to reset master.

**Comment by woctezuma** [comment]
By the way, it is:
`git fetch upstream`
instead of 
`git fetch origin upstream`

**Comment by petetnt** [comment]
Thanks for the fix! Fixed in #11. I'll do another release when #9 lands.
